### PR TITLE
Added Bitnami Django tutorial

### PR DESCRIPTION
**Pull request by vikram-bitnami**
Added Bitnami Django "hello world" tutorial. Ref #221.

- [ ] Tech writing / styleguide review - @dghale  
- [ ] Formatting review for publishing - @jmdobry

### CODE REVIEW

**Comment by vikram-bitnami** [comment]
Hi @dghale, any update or thoughts on this tutorial?

**Comment by jmdobry** [comment]
This tutorial seems a bit redundant with https://docs.bitnami.com/google/infrastructure/django/, what do you think @vikram-bitnami?

**Comment by vikram-bitnami** [comment]
@jmdobry, you're correct that the information also exists at https://docs.bitnami.com/google/infrastructure/django/. However, on that page, it's scattered across different sections and FAQs. This tutorial is a comprehensive step-by-step guide and consolidates the key information in a single place so it's easier for users to follow and get started.

**Comment by jmdobry** [comment]
@JimTravis what do you think?

**Comment by vikram-bitnami** [comment]
Hi @JimTravis, any thoughts?

**Comment by beltran-rubo** [comment]
any progress on this? @JimTravis @jmdobry @dghale

File: tutorials/get-started-bitnami-django.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+* Serve the application with Apache.
+* Configure a database (optional).
+
+## Before you begin
+
+Before starting this tutorial, ensure that you have set up a Google Cloud Platform project. You can use an existing project or [create a new project](https://console.cloud.google.com/project).
+
+## Costs
+
+The default configuration allows you to run a low-traffic Web application powered by Django using an `f1-micro` instance with a standard 10 GB persistent disk. You can customize the configuration when deploying this solution or later, although the default configuration is fine for the purposes of this tutorial.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "or later" --> "or change it later"

File: tutorials/get-started-bitnami-django.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+* Install Bitnami Django on a Google Compute Engine instance.
+* Create a "hello world" Django application.
+* Serve the application with Apache.
+* Configure a database (optional).
+
+## Before you begin
+
+Before starting this tutorial, ensure that you have set up a Google Cloud Platform project. You can use an existing project or [create a new project](https://console.cloud.google.com/project).
+
+## Costs
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "Costs" --> "Cost"

File: tutorials/get-started-bitnami-django.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+The default configuration allows you to run a low-traffic Web application powered by Django using an `f1-micro` instance with a standard 10 GB persistent disk. You can customize the configuration when deploying this solution or later, although the default configuration is fine for the purposes of this tutorial.
+
+Estimated cost for the above default configuration is $4.28 per month, based on 30-day, 24 hours per day usage in the Central US region. Sustained use discount is included.
+
+Use the [pricing calculator](https://cloud.google.com/products/calculator/) to generate a cost estimate based on your projected usage. New Google Cloud customers may be eligible for a [free trial](https://cloud.google.com/free-trial).
+
+## Install Bitnami Django on a Google Compute Engine instance
+
+In this section, you will deploy Bitnami Django on a Google Compute Engine instance.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: Suggested edit: "Deploy Bitnami Django on a Google Compute Engine instance:"

File: tutorials/get-started-bitnami-django.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+1. From the Google Cloud Platform menu, select the [Cloud Launcher](https://console.cloud.google.com/launcher).
+1. Search for "django certified by bitnami" and select the resulting `Django Certified by Bitnami` template.
+1. Review the information and costs. Click `Launch on Compute Engine` to proceed.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "costs" --> "cost"

File: tutorials/get-started-bitnami-django.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+1. Review the default zone, machine type, boot disk size and other parameters and modify as needed. Ensure that the `Allow HTTP traffic` and `Allow HTTPS traffic` boxes are checked in the firewall configuration. Click `Deploy` to proceed with the deployment.
+
+The Cloud Launcher deploys Bitnami Django on a new Google Compute Engine instance. You will be able to monitor the progress of the deployment from the [Deployment Manager](https://console.cloud.google.com/dm/deployments). Once deployed, note the public IP address of the instance and the password for the MySQL and PostgreSQL databases.
+
+## Create a "hello world" Django application
+
+In this section, you will log in to the deployed instance and create a simple Django application.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: suggested edit: "Login to the deployed instance and create a simple Django application:"

File: tutorials/get-started-bitnami-django.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+1. From the [Deployment Manager](https://console.cloud.google.com/dm/deployments), click the `SSH` button to log in to the instance over SSH.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "log in" -- > "login"

File: tutorials/get-started-bitnami-django.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+        from django.conf.urls import url
+        from django.urls import include
+        
+        urlpatterns = [
+            url(r'^helloworld/', include('helloworld.urls')),
+        ]
+
+## Serve the application with Apache
+
+Bitnami Django includes a pre-configured instance of the Apache Web server. In this section, you will configure Apache to serve the application on the standard Web server port 80.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: suggested edit: "Configure Apache to serve the application on the standard Web server port 80:"

File: tutorials/get-started-bitnami-django.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+## Install Bitnami Django on a Google Compute Engine instance
+
+In this section, you will deploy Bitnami Django on a Google Compute Engine instance.
+
+1. From the Google Cloud Platform menu, select the [Cloud Launcher](https://console.cloud.google.com/launcher).
+1. Search for "django certified by bitnami" and select the resulting `Django Certified by Bitnami` template.
+1. Review the information and costs. Click `Launch on Compute Engine` to proceed.
+1. Review the default zone, machine type, boot disk size and other parameters and modify as needed. Ensure that the `Allow HTTP traffic` and `Allow HTTPS traffic` boxes are checked in the firewall configuration. Click `Deploy` to proceed with the deployment.
+
+The Cloud Launcher deploys Bitnami Django on a new Google Compute Engine instance. You will be able to monitor the progress of the deployment from the [Deployment Manager](https://console.cloud.google.com/dm/deployments). Once deployed, note the public IP address of the instance and the password for the MySQL and PostgreSQL databases.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "will be able to" --> "can"

File: tutorials/get-started-bitnami-django.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+      }
+    }
+
+## Cleaning up
+
+After you have finished this tutorial, you can remove the resources you created on Google Cloud Platform so you aren't billed for them any longer. You can delete the resources individually, or delete the entire project.
+
+### Deleting the project
+
+Visit the [Resource Manager](https://console.cloud.google.com/cloud-resource-manager). Select the project you used for this tutorial and click `Delete`. Once deleted, you will not be able to reuse the project ID.
```
Comment (chenopis) [inline tutorials/get-started-bitnami-django.md]: "you will not be able to" --> "you cannot"

**Comment by vikram-bitnami** [comment]
@chenopis, please see my changes. Thanks.

**Comment by vikram-bitnami** [comment]
Could you please merge the pull request @JimTravis @jmdobry @dghale @chenopis? Thank you.

**Pull request merged**